internal/storage/types/ssblob/ssblob_test.go

```diff
@@ -0,0 +1,37 @@
+package ssblob_test
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"gotest.tools/v3/assert"
+
+	"github.com/artefactual-sdps/enduro/internal/storage/types/ssblob"
+)
+
+// fix tests to work for archivematica storage service. Add proper mocks etc for this I am not sure how exactly to go about it.
```

Comment:
You can create a [httptest.Server](https://pkg.go.dev/net/http/httptest#NewServer) where `yourHandlerFunction` returns some data and validates the client request. E.g.:

```go
srv := httptest.NewServer(http.HandlerFunc(yourHandlerFunction))
```
    
Use `srv.URL` to configure the new bucket.